### PR TITLE
feat: Create beautiful but functionless login button

### PR DESCRIPTION
**Pull request by david-code**
## Description

This button is just an experiment to learn how to create components, install nativebase and customize the theme. The next steps will be to update this button so it actually returns logs the user in.

I've temporarily commented out the map view. As part of this feature, I'd like to work out how to figure out navigation, so the user is redirected to the map view after being logged in.

Making this a separate PR to check out the following things:
- Does the theme setup look reasonable?
- folder structure for components

Related to #44

### CODE REVIEW

File: dev-client/App.tsx
Comment (david-code) [inline dev-client/App.tsx:29]: Commented this out for now. Will implement some kind of routing as part of this feature and uncomment before we merge into main.

File: dev-client/package.json
Comment (david-code) [inline dev-client/package.json:18]: As per suggested in the [nativebase docs](https://docs.nativebase.io/install-rn#h3-install-dependencies)

**Comment by david-code** [comment]
Question about running the Apple `pod install` or whatever needed to complete the installation of a new package with native integration: how/when do we want to do this?

**Review by shrouxm — APPROVED** [review]
LGTM! we can change the folder structure if it doesn't seem to be working, but that seems reasonable

**Comment by shrouxm** [comment]
re `pod install`, i'm really not sure, i've been leaning on @paulschreiber for iOS help so far but i think it's your call at this point, i just don't have any way to experiment and form opinions

**Pull request merged**
